### ISSUE TITLE
Add google_visualr to the Gemfile

### ISSUE THREAD

**Issue by athityakumar** [Shekharrajak/daru_examples_io_view_rails#1]
After running `bundle` and `rails server`, I get `-- cannot require google_visualr` error. Adding it to the Gemfile seems to fix this though.

**Comment by Shekharrajak** [comment]
fixed through https://github.com/Shekharrajak/daru-view/pull/46

**Comment by athityakumar** [comment]
Acknowledged, that seems to have been fixed now. Let's also fix the `ActionController::API:Class` error and then close this issue? 😄

**Comment by Shekharrajak** [comment]
>  Let's also fix the ActionController::API:Class error

Where did you find this error?

**Comment by Shekharrajak** [comment]
Okay cool ! the error 
```
Puma caught this error: undefined method `helper_method' for ActionController::API:Class (NoMethodError)
```

**Comment by athityakumar** [comment]
Fixed, as per commit 07cc873a0830e81ad8d4ecddda40b1b9cc6eef9b. Closing this issue.
